docs/source/deployment/netlify.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Deploy a fullstack Apollo app with Netlify
+sidebar_title: Netlify
+description: How to run your API using Netlify Functions on AWS Lambda
+---
+
+[Netlify functions](https://www.netlify.com/docs/functions/), is a new way to deploy server-side code to AWS Lambda alongside your frontend, right from GitHub. In this guide, I'll show you why Netlify's functions is a great way to run your whole Apollo app.
+
+1. Use the same platform to deploy your frontend and API code.
+2. Deploy previews of both your frontend and API on every pull request.
+3. No need to worry about scaling containers due to a serverless architecture.
+
+We'll set up a "Hello World" API and frontend for an Apollo app on Netlify. Let's get started!
+
+## Download and Run the Netlify Starter Kit
+
+Download and run the [official Netlify create-react-app and Lambda starter kit](https://github.com/netlify/create-react-app-lambda). This will set us up with a base for our frontend and API code.
+
+
+```bash
+git clone https://github.com/netlify/create-react-app-lambda.git
```

Comment:
Could this perhaps be easier to digest using [`npm init`'s `initializer`](https://docs.npmjs.com/cli/init#examples)?

```bash
npm init react-app-lambda
```

Reply:
Hmm, I'm not sure how this would currently work since [`create-create-app-lambda` isn't an npm package](http://npm.im/create-react-app-lambda), merely a Git repo.  Additionally, I don't believe that initializers work in any sort of automatic cloning way?  A script would have to be specified as its main `bin` which did the cloning — right?  For example, [`create-esm`'s main `bin`](https://github.com/standard-things/create-esm/blob/c2e1ff3e19cf2ec478bb93ed857d6b0e52cb6ff0/package.json#L9) is the `create-esm` script.  That script, [does a bunch of things](https://github.com/standard-things/create-esm/blob/c2e1ff3e19cf2ec478bb93ed857d6b0e52cb6ff0/src/create-esm.js) to make it work.  Unless I'm missing something?

Reply:
Good point @abernix about `create-react-app-lambda` not being an npm package, not sure if it would work.

I assumed it would because the repo is based on CRAv2 and CRA recommends using the [`npm init react-app` pattern](https://github.com/facebook/create-react-app#npm)

I didn't realized it until now; that will require npm 6, which obviously isn't a minimum for these docs.